DelvUI/Config/Attributes/ConfigTypeAttributes.cs

```diff
@@ -6,14 +6,19 @@ namespace DelvUI.Config.Attributes
     public class ConfigAttribute : Attribute
     {
         public bool isMonitored = false;
+        public bool separator = false;
     }
 
     [AttributeUsage(AttributeTargets.Field)]
     public class CheckboxAttribute : ConfigAttribute
     {
         public string friendlyName;
 
-        public CheckboxAttribute(string friendlyName) { this.friendlyName = friendlyName; }
+        public CheckboxAttribute(string friendlyName)
+        {
+            this.friendlyName = friendlyName;
+            separator = false;
```

Comment:
You don't need to do this in every constructor when the base class is already setting the initial value to false